lesson_04/homework_04.py

```diff
@@ -23,48 +23,51 @@
 # task 01 ==
 """ Дані у строці adwentures_of_tom_sawer розбиті випадковим чином, через помилку.
 треба замінити кінець абзацу на пробіл .replace("\n", " ")"""
-
+task_01 = adwentures_of_tom_sawer.replace("\n ", " ")

```

Comment:
Linter shows me this:
```
./bin/lint ./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:1:1: D100 Missing docstring in public module
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:8:80: E501 line too long (80 > 79 characters)
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:13:80: E501 line too long (82 > 79 characters)
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:22:1: E266 too many leading '#' for block comment
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:24:80: E501 line too long (82 > 79 characters)
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:26:43: Q000 Double quotes found but single quotes preferred
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:26:50: Q000 Double quotes found but single quotes preferred
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:30:27: Q000 Double quotes found but single quotes preferred
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:30:35: Q000 Double quotes found but single quotes preferred
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:34:27: Q000 Double quotes found but single quotes preferred
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:34:33: Q000 Double quotes found but single quotes preferred
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:38:41: Q000 Double quotes found but single quotes preferred
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:49:24: Q000 Double quotes found but single quotes preferred
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:55:59: Q000 Double quotes found but single quotes preferred
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:66:30: Q000 Double quotes found but single quotes preferred
./T_HOMETASK/Lesson_04/04_Natalia_Nzarenko.py:69:80: E501 line too long (83 > 79 characters)
```
Next time please pay attention to linter warnings and try to fix them